tests/airgapped/README.md

```diff
@@ -27,8 +27,9 @@ You can run the script that will spin up an airgapped microk8s cluster with:
 ```bash
 ./tests/airgapped/airgap.sh \
   --node-name airgapped-microk8s \
-  --microk8s-channel 1.24/stable \
-  --bundle-path releases/latest/edge/bundle.yaml
+  --microk8s-channel 1.25-strict/stable \
+  --bundle-path releases/1.8/stable/kubeflow/bundle.yaml \
+  --juju-channel 3.1/stable
```

Comment:
question: Should we mention why juju 3.1 is needed?

Reply:
juju 3.1 is the supported version for CKF 1.8, you can see that in the [dependency versions matrix](https://charmed-kubeflow.io/docs/supported-versions)
Do you think I should add a note there? imo the team always has context on this and this directory is mainly for the developers :)